src/sections/StoreAddress.scss

```diff
@@ -47,13 +47,6 @@ div.nfd-ecommerce-store-address-form {
     }
   }
 
-  @media (max-width: 960px) {
```

Comment:
Are we sure this is safe? Can you try the UI at around 800px-850px and see if it doesn't break?

Reply:
Yes tested. Its working fine